src/runtime_src/core/pcie/driver/linux/include/version.h

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
Isn't this file generated during building XRT? Do we want to check in a static version of it?

Reply:
Opps, we definitely don't want this file.  Good catch.

Reply:
Thanks! I removed it.